android/src/org/coolreader/CoolReader.java

```diff
@@ -741,7 +741,17 @@ private boolean processIntent(Intent intent) {
 			if (uri != null) {
 				fileToOpen = filePathFromUri(uri);
 			}
+		} else if ("org.coolreader.PAGE_UP".equals(intent.getAction())) {
+			mReaderView.onCommand(ReaderCommand.DCMD_PAGEUP, 0, null);
+			return true;
+		} else if ("org.coolreader.PAGE_DOWN".equals(intent.getAction())) {
+			mReaderView.onCommand(ReaderCommand.DCMD_PAGEDOWN, 0, null);
+			return true;
+		} else if ("org.coolreader.TTS_PLAY".equals(intent.getAction())) {
+			mReaderView.onCommand(ReaderCommand.DCMD_TTS_PLAY, 0, null);
+			return true;
```

Comment:
I did it via:

for (ReaderAction ra: ReaderAction.AVAILABLE_ACTIONS) {
			String acname = "org.knownreder." + StrUtils.getNonEmptyStr(ra.id, true);
			if (acname.equals(intentAction)) {
				mReaderView.onCommand(ra.cmd, ra.param, null);
				return true;
			}
		}

I think this is more convient solution